src/server/rest/v1/service/RegistrationTokenService.ts

```diff
@@ -249,4 +251,47 @@ export default class RegistrationTokenService {
     res.json(registrationTokens);
     next();
   }
+
+  static async handleGetRegistrationToken(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    // Check auth
+    if (!Authorizations.canReadRegistrationToken(req.user)) {
+      // Not Authorized!
+      throw new AppAuthError({
+        errorCode: HTTPAuthError.ERROR,
+        user: req.user,
+        action: Action.READ, entity: Entity.TOKEN,
+        module: MODULE_NAME, method: 'handleGetRegistrationToken'
+      });
+    }
+    const filteredRequest = RegistrationTokenSecurity.filterRegistrationTokenRequest(req.query);
+    // Check User
+    let userProject: string[] = [];
+    if (Authorizations.canListUsers(req.user)) {
+      userProject = [ 'createdBy.name', 'createdBy.firstName', 'lastChangedBy.name', 'lastChangedBy.firstName' ];
+    }
+    // Get the token
+    const registrationToken = await RegistrationTokenStorage.getRegistrationToken(req.user.tenantID,
+      filteredRequest.id,
```

Comment:
If it's a GET rename id -> ID

Reply:
I changed the `filterRegistrationTokenRequest` with `filterRegistrationTokenByIDRequest` so only the id is returned. it's already handled as "ID" param